app/assets/stylesheets/task.scss

```diff
@@ -21,3 +30,17 @@
 
   i { line-height: 1rem; }
 }
+
+.tag {
+  @extend .chip;
+  background-color: #E6E18F;
+  color: #92977E;
```

Comment:
Color literals like `#92977E` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#92977E` should be written as `#92977e`